### PR TITLE
Pass source id instead of source to resolvedSourceID

### PR DESCRIPTION
**Pull request by ebb-tide**

### CODE REVIEW

File: ui/src/shared/containers/SetActiveSource.tsx [hunk 1]
```diff
@@ -52,14 +52,14 @@ class SetActiveSource extends PureComponent<Props> {
 
     const querySourceID = readQueryParams().sourceID
 
-    let resolvedSourceID
+    let resolvedSourceID: string
```
Comment (alexpaxton) [inline ui/src/shared/containers/SetActiveSource.tsx]: TYPES! Huzzah

File: ui/src/shared/containers/SetActiveSource.tsx [hunk 2]
```diff
@@ -52,14 +52,14 @@ class SetActiveSource extends PureComponent<Props> {
...
 
     if (sources.find(s => s.id === activeSourceID)) {
       resolvedSourceID = activeSourceID
     } else if (sources.find(s => s.id === querySourceID)) {
       resolvedSourceID = querySourceID
     } else if (sources.length) {
-      resolvedSourceID = sources[0]
+      resolvedSourceID = sources[0].id
```
Comment (alexpaxton) [inline ui/src/shared/containers/SetActiveSource.tsx]: Does this need a guard?

File: ui/src/shared/containers/SetActiveSource.tsx (hunk 1 above)
Comment (ebb-tide) [reply ui/src/shared/containers/SetActiveSource.tsx]: I KNOWWW! :)

File: ui/src/shared/containers/SetActiveSource.tsx (hunk 2 above)
Comment (ebb-tide) [reply ui/src/shared/containers/SetActiveSource.tsx]: good idea, thanks

**Pull request merged**